pkg/cli/cmds/secrets_encrypt.go

```diff
@@ -84,6 +84,13 @@ func NewSecretsEncryptCommands(status, enable, disable, prepare, rotate, reencry
 						Destination: &ServerConfig.EncryptSkip,
 					}),
 			},
+			{
+				Name:           "rotate-keys",
+				Usage:          "(Experimental) Add, rotate and rencryption with a new encryption key",
```

Comment:
convention is for this to be lowercase
```suggestion
				Usage:          "(experimental) Dynamically add a new secrets encryption key and re-encrypt secrets",
```